lib/widgets/NextButton.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:random_run/theme.dart' as T;
+
+class NextButton extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return ButtonTheme(
+      minWidth: T.Spacing.xlarge,
+      height: 70.0,
```

Comment:
any other way we could do this instead of setting specific values for height/widths? maybe just adding some padding would be fine?